src/main/resources/Diagram/DiagramTranslations.fr.xml

```diff
@@ -57,19 +57,19 @@ diagram.livetable.doc.location=Emplacement
 diagram.livetable.emptyvalue=-
 
 # Diagram Macro
-rendering.macro.diagram.description=Affiche un diagramme.
+rendering.macro.diagram.description=Affiche une diagramme.
 rendering.macro.diagram.name=Diagramme
-rendering.macro.diagram.parameter.cached.description=Afficher le diagramme sous la forme d'une image mise en cache (plus rapide) plutôt que d'un dessin vectoriel (plus lent).
-rendering.macro.diagram.parameter.cached.name=Mise en cache
-rendering.macro.diagram.parameter.reference.description=rendering.macro.diagram.parameter.reference.description=La référence vers la page qui contient le diagramme à afficher. Ex : pour "http:~/~/.../Diagram/Mon%20Diagramme/", saisir "Diagram.Mon Diagramme.WebHome". Si ce champ est laissé vide, la macro recherche une page descendante nommée "Diagramme" et, en cas d'absence, affiche un formulaire de création de diagramme.
-rendering.macro.diagram.parameter.reference.name=Référence
-diagram.macro.create=Créer un diagramme
-diagram.macro.createNotAllowed=Vous n'êtes pas autorisé(e) à créer ce diagramme.
-diagram.macro.edit=Modifier le diagramme
-diagram.macro.invalidReference=La page spécifiée n'est pas un diagramme.
-diagram.macro.view=Voir le diagramme
-diagram.macro.viewNotAllowed=Vous n'êtes pas autorisé(e) à voir ce diagramme.
+rendering.macro.diagram.parameter.cached.description=Afficher une image d'une diagramme cachée (plus rapide) au lieu de rendre la diagramme en live (plus lent).
+rendering.macro.diagram.parameter.cached.name=Caché
+rendering.macro.diagram.parameter.reference.description=rendering.macro.diagram.parameter.reference.description=La référence vers la page qui contient la diagramme à afficher. Ex. pour "http:~/~/.../Diagram/My%20Diagram/", rentrez "Diagram.My Diagram.WebHome". Si ce champ est laissé vide, la macro cherche pour une page-enfant nommée "Diagramme" et, si pas trouvée, affiche un formulaire qui permet la création d'une diagramme.
+rendering.macro.diagram.parameter.reference.name=Reférence
+diagram.macro.create=Créer diagramme
+diagram.macro.createNotAllowed=Vous n'êtes pas autorisé à créer cette diagramme.
+diagram.macro.edit=Modifier diagramme
+diagram.macro.invalidReference=La page spécifiée n'est pas une diagramme.
+diagram.macro.view=Voir diagramme
+diagram.macro.viewNotAllowed=Vous n'êtes pas autorisé à voir cette diagramme.
 
 # Diagram editor
-diagram.editor.saveAsImageAttachmentError=Erreur à l'enregistrement du diagramme comme un attachement.</content>
+diagram.editor.saveAsImageAttachmentError=Erreur à l'enregistrement du diagramme comme une attachement.</content>
```

Comment:
Are you sure about this? Isn't `un attachment` or `une piece jointe`?

Reply:
You're right Alex, it's just I re-synced this branch with master, hence erasing previous translation fixes wrongly pushed to this branch initially. We can fix the translation issues either in master or in a dedicated branch, not in this one, right?

Reply:
Sure, as you wish! Thanks!